src/main/java/com/k3ntako/HTTPServer/controllers/GetTextFileContent.java

```diff
@@ -0,0 +1,28 @@
+package com.k3ntako.HTTPServer.controllers;
+
+import com.k3ntako.HTTPServer.ControllerInterface;
+import com.k3ntako.HTTPServer.FileIOInterface;
+import com.k3ntako.HTTPServer.RequestInterface;
+import com.k3ntako.HTTPServer.Response;
+
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.Path;
+
+public class GetTextFileContent implements ControllerInterface {
+  private FileIOInterface fileIO;
+
+  public GetTextFileContent(FileIOInterface fileIO) {
+    this.fileIO = fileIO;
+  }
+
+  public Response getResponse(RequestInterface request) throws IOException {
+    Path path = FileSystems.getDefault().getPath("./data/simple_post.txt");
```

Comment:
can it get file content of any text file, not just that one?

Reply:
Because this is a specific endpoint, I thought this controller would be the place to put the exact file path. Because this file implements the `ControllerInterface`, I don't think it should take in unique arguments such as a file path. 

I did try to abstract the file retrieval by creating the `FileIO` class.

Does that make sense?